generator/internal/api/skip.go

```diff
@@ -20,29 +20,47 @@ import (
 )
 
 func SkipModelElements(model *API, options map[string]string) {
-	ids, ok := options["skipped-ids"]
-	if !ok {
-		return
+	included_ids, included_ok := options["included-ids"]
+	skipped_ids, skipped_ok := options["skipped-ids"]
+	if included_ok && skipped_ok {
+		panic("Both `included-ids` and `skipped-ids` set. Only set one.")
 	}
```

Comment:
Seems like we should change the return type from this function to be `error` instead of panicking.

Reply:
Or use `slog.Fatal()` which I believe we use elsewhere.  But I would prefer returning an error.